src/backend/renderer/multigpu/mod.rs

```diff
@@ -50,6 +50,7 @@ use std::{
     rc::Rc,
 };
 
+use crate::backend::renderer::gles::Capability;
```

Comment:
This causes the multigpu code to depend on types in gles. Capabilities probably could be put to a later time here since we probably want more broad capabilities that are renderer generic.